app/api/ada/index.js

```diff
@@ -1062,7 +1062,8 @@ export default class AdaApi {
       );
 
       Logger.debug('AdaApi::restoreWallet success');
-      return _createWalletFromServerData(adaWallet);
+      const account = getCurrentCryptoAccount();
```

Comment:
@sarveshwar-singh this code change to which fix is related?

Reply:
FYI this will also have a merge conflict with #567

Reply:
@nicarq 
Regarding: https://app.clubhouse.io/emurgo/story/1382/fix-top-bar-on-new-wallet-creation